### PR TITLE
issue #528: chore (build): address scons file inconsistencies vis a vis ubuntu

### PR DESCRIPTION
**Pull request by tst-basilhuffman**
addresses issue #528

### CODE REVIEW

File: earth_enterprise/src/SConstruct [hunk 1]
```diff
@@ -150,16 +150,27 @@ else:
   python_include = '/usr/include/python%s' % python_major_version
 
 # setup platform specific operations
-is_hardy = True
+is_min_ubuntu = True
 # if non-zero return status it is not hardy
```
Comment (tst-jblair) [inline earth_enterprise/src/SConstruct]: This comment block is out of sync with the code, now. Presumably it should just be removed?

Comment (tst-basilhuffman) [reply earth_enterprise/src/SConstruct]: I dont follow

Comment (tst-basilhuffman) [reply earth_enterprise/src/SConstruct]: Sorry Jim, I misunderstood your comment. No issues, will change.

**Comment by tst-mswartz** [comment]
This looks ok to me ... As an aside though (and I know we don't support Mint) but my `lsb_release` returns 18.2

**Pull request merged**